app/components/Project/ProjectForm.tsx

```diff
@@ -99,38 +87,28 @@ const ProjectForm: React.FC<Props> = (props) => {
         operatorTradeName: {
           type: "string",
         },
-        fundingStreamId: {
-          type: "number",
-          title: "Funding Stream ID",
-          default: undefined,
-          anyOf: query.allFundingStreams.edges.map(({ node }) => {
-            return {
-              type: "number",
-              title: node.name,
-              enum: [node.rowId],
-              value: node.rowId,
-            };
-          }),
-        },
-        fundingStreamDescription: {
-          type: "string",
-        },
         fundingStreamRfpId: {
           type: "number",
           title: "Funding Stream RFP ID",
           default: undefined,
-          anyOf: rfpOptions.map(({ node }) => {
-            return {
-              type: "number",
-              title: node.year.toString(),
-              enum: [node.rowId],
-              value: node.rowId,
-            };
-          }),
+          parent: {
+            entityType: "allFundingStreams",
+            displayField: "name",
+            placeholder: "Select a Funding Stream",
+            label: "Funding Stream"
+          },
+          child: {
+            entityType: "allFundingStreamRfps",
+            displayField: "year",
+            placeholder: "Select a Funding Stream RFP Year",
+            label: "Funding Stream RFP"
+          },
+          foreignKey: "fundingStreamId",
+          query: query,
```

Comment:
I'd rather keep our JSON schema a valid entity. This is the reason why we're using `anyOf` instead of `enumNames` for select components. Adding custom fields to our JSON schema will harm code readability (as one would assume that this is an actual JSON schema, but it isn't), and might prevent use of this schema with other libraries (e.g. if we wanted to so server-side validation of the change before committing)